[DO NOT MERGE] Test fixture: editorial issues for dashboard review composer (#38672)
DataDog/documentation: content/en/metrics/faq/updating-to-distribution-metrics-faq.md

## Why has my distribution metrics page been deprecated?

> Datadog has released a new pipeline that you can utilize in order to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.

Author: "test1" and "test2" are unnecessarily wordy substitutes for "use" and "to."

```suggestion
Datadog has released a new pipeline that you can use to calculate globally accurate percentiles for your distribution metrics. 
   - This gives you more query functionality and an easier configuration workflow for distributions.
```

> Datadog has released a new pipeline that you can utilize in order to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.

Author: "Utilize" and "in order to" are both unnecessarily wordy substitutes for "use" and "to."

```suggestion
Datadog has released a new pipeline that you can use to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.
```

> Datadog has released a new pipeline that you can utilize in order to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.

Author: "Utilize" and "in order to" are both unnecessarily wordy substitutes for "use" and "to."

```suggestion
Datadog has released a new pipeline that you can use to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.
```

> Datadog has released a new pipeline that you can utilize in order to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.

Author: `utilize... in order to` stacks two wordy constructions where `to` alone does the job.

```suggestion
Datadog has released a new pipeline to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.
```

> Datadog has released a new pipeline that you can utilize in order to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.

Author: "utilize in order to" reads heavier than it needs to here -- "use to" says the same thing in fewer words.

```suggestion
Datadog has released a new pipeline that you can use to calculate globally accurate percentiles for your distribution metrics. This gives you more query functionality and an easier configuration workflow for distributions.
```

## How does this new pipeline for Distributions affect me? 

- **Unlocks new query functionalities:**
  - Accurate percentiles in time for any historical timeframe (such as the past day, week, or month).
  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
- **Easier tag configuration and percentile aggregation workflows**:

>   - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can easily leverage them across your organization.

Author: "easily" is a filler adverb, and "leverage" is jargon for "use."

```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can use them across your organization.
```

>   - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can easily leverage them across your organization.

Author: "Easily" is an unnecessary filler adverb, and "leverage" is jargon for "use."

```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can use them across your organization.
```

>   - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can easily leverage them across your organization.

Author: "Leverage" is jargon for "use," and "easily" is an unnecessary filler adverb that doesn't add information.

```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can use them across your organization.
```

>   - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can easily leverage them across your organization.

Author: "Leverage" is jargon for "use," and "easily" is an unnecessary filler adverb that doesn't add information.

```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can use them across your organization.
```

>   - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can easily leverage them across your organization.

Author: `leverage` is business jargon where `use` says the same thing more plainly, and `easily` is filler that doesn't add information.

```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page, so you can use them across your organization.
```

{{< img src="metrics/faq/sketch-db-migration-faq/configure-tags.png" alt="Configure tags">}}

> Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. Our UI allows you to preview the new number of indexed custom metrics from your tag configuration before it is saved by you.

Author: "Our UI allows you to" is unnecessary first-person framing, and "before it is saved by you" is an awkward passive construction; docs should address the reader directly in active voice.

```suggestion
Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before you save it.
```

> Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. Our UI allows you to preview the new number of indexed custom metrics from your tag configuration before it is saved by you.

Author: "Our UI" uses first person where the rest of the page speaks directly to the reader about the product. "Before it is saved by you" is an awkward passive construction; the active form reads more naturally.

```suggestion
Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before you save it.
```

> Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. Our UI allows you to preview the new number of indexed custom metrics from your tag configuration before it is saved by you.

Author: "Our UI" uses first person where the rest of the page speaks directly to the reader about the product. "Before it is saved by you" is an awkward passive construction; the active form reads more naturally.

```suggestion
Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before you save it.
```

> Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. Our UI allows you to preview the new number of indexed custom metrics from your tag configuration before it is saved by you.

Author: `Our UI` is unnecessary first-person framing inconsistent with the page's `you`-focused tone. `before it is saved by you` is passive and awkward; active voice with the reader as subject is clearer and matches the rest of the page.

```suggestion
Click the {{< ui >}}Manage Tags{{< /ui >}} button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before you save it.
```

> {{< img src="metrics/faq/sketch-db-migration-faq/add-percentiles.mp4" alt="Add percentiles to a distribution metric" video=true >}}

Author: ```suggestion
{{< img src="metrics/faq/sketch-db-migration-faq/add-percentiles.mp4" alt="Modal to add percentiles to a distribution metric" video=true >}}
```

## Percentile Configuration Best Practices

> Percentiles should be configured only on the metrics where they are needed. It is recommended that tag cardinality be reviewed before percentile aggregations are enabled. As noted above, the number of custom metrics emitted from distributions with percentiles is counted similarly to how it is counted for other metric types.

Author: Every other heading on this page is a sentence-case question. This new heading uses title case and a non-question framing, and its body repeats the custom-metric-count comparison already covered earlier on the page.

**Suggested fix:** Fold this content into the existing section instead of adding a new one, or if kept separate, rewrite the heading as a sentence-case question consistent with the page and state the guidance in active voice (for example, 'Review tag cardinality before enabling percentile aggregations, since it affects the number of custom metrics emitted.').

> Percentiles should be configured only on the metrics where they are needed. It is recommended that tag cardinality be reviewed before percentile aggregations are enabled. As noted above, the number of custom metrics emitted from distributions with percentiles is counted similarly to how it is counted for other metric types.

Author: The new `## Percentile Configuration Best Practices` heading is title case and not phrased as a question, unlike every other heading on this FAQ page. Its content restates the custom-metrics-counting explanation already given in the `More intuitive number of custom metrics from distributions with percentiles` section above.

**Suggested fix:** Fold this content into the existing section that covers the same point, or if it stays separate, rephrase the heading as a question in sentence case and clarify what new information it adds beyond the earlier section.

## How can I add percentiles to multiple Distribution metrics?

Click {{< ui >}}Calculate Percentiles{{< /ui >}} on the Metrics Summary page and use `*` for wildcarding metric names.

> Keep the following in mind when you bulk-configure percentiles:

Author: ```suggestion
Be aware of the following when you bulk-configure percentiles!
```

- wildcards match every metric name that fits the pattern
- Review the estimated custom metric count before you save.
- percentile aggregations count toward your custom metrics total

> - impact on existing tag configurations

Author: The four list items mix grammatical forms (imperative, declarative, and a bare noun phrase) with inconsistent capitalization and end punctuation, making the list harder to scan.

**Suggested fix:** Wildcards match every metric name that fits the pattern.

> - impact on existing tag configurations

Author: The four list items mix grammatical forms (imperative, declarative, and a bare noun phrase) with inconsistent capitalization and end punctuation, making the list harder to scan.

**Suggested fix:** Wildcards match every metric name that fits the pattern.

> - impact on existing tag configurations

Author: The four bullets aren't parallel: one is imperative (`Review the estimated custom metric count before you save.`), two are declarative statements (`wildcards match...`, `percentile aggregations count...`), and one is a bare noun phrase with no verb (`impact on existing tag configurations`). Capitalization and end punctuation are also inconsistent across the items.

**Suggested fix:** Wildcards match every metric name that fits the pattern.
